pset-iterators.js

```diff
@@ -115,10 +131,17 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const presidentsBetween1850and1900 = presidents.filter(function (between1850And1900) {
+  return (between1850And1900.left_office) ? parseInt(between1850And1900.took_office.substring(0, 4)) >= 1850 && parseInt(between1850And1900.left_office.substring(0, 4)) < 1900 : false;
+}).map(function (president) {
+  return president.president
+})
+
+console.log(presidentsBetween1850and1900)
```

Comment:
@OcielC Good job here on this solution. I see you're using a filter method to return the president's who served between1850and1900. Good job using the ternary operator for this solution.
Suggestions
- For this example, you don't need the map method, since you're returning an array of objects the filter method will be sufficient.
- also for the ternary object here another way you could've written the statement.
```
if (between1850And1900.left_office){
  return parseInt(between1850And1900.took_office.substring(0, 4)) >= 1850 && 
  parseInt(between1850And1900.left_office.substring(0, 4)) < 1900;
}
else{
  return false;
}
```
Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L120)